django_redis/serializers/crypted.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
+
+from .pickle import PickleSerializer
+
+from cryptography.fernet import Fernet
+from django.conf import settings
```

Comment:
Yes, there is such a thing ... We must be clear at itself in code.
In any case, it does not matter: the project author sees no sense in this commit.